pages.bn/common/ack.md

```diff
@@ -0,0 +1,36 @@
+# ack
+
+> একটি গ্রেপের মত খোঁজ টুল, ডেভেলপারদের জন্য অপটিমাইজড করা।
```

Comment:
There is a missing line here and the search pattern placeholders aren't translated. Please update this file.